### PR TITLE
Onboarding/fetch gozturk7

### PR DESCRIPTION
**Pull request by GOZTURK7**

### CODE REVIEW

File: packages/ui/src/examples/E2_FetchWithStrapiRequest.tsx [hunk 1]
```diff
@@ -2,16 +2,47 @@
...
+    }
+    axios
+      .get(API_URL, { headers })
+      .then(response => setBlogs(response.data.data))
+      .catch(err => console.log(err))
+  }
 
   useEffect(() => {
     // TODO: fetch blogs with fetcher by using the API_URL and TOKEN
     // Remember that fetcher is a wrapper around axios that adds the token and api url to the request
```
Comment (7alip) [inline packages/ui/src/examples/E2_FetchWithStrapiRequest.tsx:36]: My mistake, I should have update the comments here. I expected to see `strapiRequest` also in this example, but it seems the comments about 'fetcher' made you confused.

You already used `strapiRequest` in the other example, so it's not problem

File: packages/ui/src/examples/E3_FetchWithFilters.tsx [hunk 2]
```diff
@@ -2,24 +2,66 @@
...
+  return (
+    <Box>
+      <label htmlFor="search">Serach Blog By Title</label>
+      <input
+        name="search"
+        id="search"
+        type="text"
+        value={titleFilter}
+        onChange={inputHandler}
+      ></input>
```
Comment (7alip) [inline packages/ui/src/examples/E3_FetchWithFilters.tsx:61]: For challenging, you could use Strapi form components here.

**Review by 7alip — APPROVED** [review]
Thank you, excellent work. You can now close the PR.

**Comment by GOZTURK7** [comment]
many thanks.

**Pull request closed without merging**